build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts

```diff
@@ -133,8 +134,17 @@ fun Project.extractCiData() {
             tag("CODEQL")
         }
     }
+    if (isTeamCity && !isKillLeakingProcessesStep()) {
+        buildScan {
+            buildScanPublished {
+                println("##teamcity[buildStatus text='{build.status.text}: ${this.buildScanUri}']")
```

Comment:
This shows the last interesting build scan when we have more than one. We do this for example for Gradleception. Though I think this is fine.